stl/inc/queue

```diff
@@ -135,7 +135,7 @@ public:
 #endif // _HAS_CXX23
 
     template <class... _Valty>
-    _CONTAINER_EMPLACE_RETURN emplace(_Valty&&... _Val) {
+    _ADAPTOR_EMPLACE_RETURN emplace(_Valty&&... _Val) {
 #if _HAS_CXX17
         return c.emplace_back(_STD forward<_Valty>(_Val)...);
 #else // ^^^ _HAS_CXX17 / !_HAS_CXX17 vvv

```

Comment:
I understand you avoid the void return for no reason, but here it could save a preprocessor branch, why not do that?

Reply:
Oh, actually the void return would enforce the void return type in C++4 mode, which is certainly not desired.